mix.exs

```diff
@@ -66,7 +73,13 @@ defmodule DigitalCollex.MixProject do
     [
       "ecto.setup": ["ecto.create", "ecto.migrate", "run priv/repo/seeds.exs"],
       "ecto.reset": ["ecto.drop", "ecto.setup"],
-      test: ["ecto.create --quiet", "ecto.migrate", "test"]
+      "assets.compile": &compile_assets/1,
+      test: [
+        "assets.compile",
```

Comment:
I am not certain that I prefer that this be added here (it's likely going to slow running the tests for suites which do not require that Webpack assets be compiled). I've attempted to add a separate alias which also invokes "test" (both `test.features` and `featuretests`), but received:

```
** (Mix) "mix test" is running in the "dev" environment. If you are running tests from within another command, you can either:

  1. set MIX_ENV explicitly:

      MIX_ENV=test mix test.another

  2. set the :preferred_cli_env for a command inside "def project" in your mix.exs:

      preferred_cli_env: ["test.another": :test]
```

Reply:
@jrgriffiniii it seems like option 2 would work. did that not work?